README.md

```diff
@@ -33,6 +33,13 @@ Install `insanely-fast-whisper` with `pipx` (`pip install pipx` or `brew install
 ```bash
 pipx install insanely-fast-whisper
 ```
+
+You also need to have ffmpeg installed; 
+
+```bash
+brew install ffmpeg
```

Comment:
this is only for mac, it would be nice to explain it and other options

Reply:
I would assume `apt install ffmpeg` for Debian-based operating systems and `winget install ffmpeg` for Windows as well.